velox/functions/prestosql/types/fuzzer_utils/tests/HyperLogLogInputGeneratorTest.cpp

```diff
@@ -53,4 +54,29 @@ TEST_F(HyperLogLogInputGeneratorTest, generate) {
   }
 }
 
+TEST_F(HyperLogLogInputGeneratorTest, generateP4HyperLogLog) {
```

Comment:
nit: I originally thought we would just use HyperLogLogInputGenerator for P4HyperLogLog too, so I suggested moving this unit test to HyperLogLogInputGeneratorTest.cpp. But it's good that we still keep a separate P4HyperLogLogInputGenerator. Then it's better to still have this test case in P4HyperLogLogInputGeneratorTest.cpp. Sorry for causing confusion.